synthesis/codon/codon.go

```diff
@@ -150,17 +150,30 @@ func (codonTable Table) OptimizeTable(sequence string) Table {
 }
 
 // GetCodingRegions is a helper function to pull coding regions out of an Sequence as input for optimizing codon tables.
-func GetCodingRegions(sequence poly.Sequence) string {
+func GetCodingRegions(sequence poly.AnnotatedSequence) (string, error) {
```

Comment:
Here's the function itself that says I'm failing to implement feature interface.